rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -97,4 +110,34 @@ impl Run {
     pub fn attempt_count() -> u32 {
         LIVESPLIT_STATE.lock().unwrap().timer.run().clone().attempt_count()
     }
+    pub fn save_splits(path: String) -> Result<(), SplitsSaveError> {
+        let state = LIVESPLIT_STATE.lock().unwrap();
+        let mut timer = state.timer.clone();
```

Comment:
No need to clone. No need for `mut`.

Reply:
```rust
error[E0507]: cannot move out of dereference of `MutexGuard<'_, rebo::livesplit::LiveSplit>`
   --> src/threads/ue/rebo/livesplit.rs:115:21
    |
115 |         let timer = state.timer;
    |                     ^^^^^^^^^^^ move occurs because value has type `livesplit_core::Timer`, which does not implement the `Copy` trait
    |
help: consider borrowing here
    |
115 |         let timer = &state.timer;
    |                     +
help: consider cloning the value if the performance cost is acceptable
    |
115 |         let timer = state.timer.clone();
    |                                ++++++++
```